### PR TITLE
cm14.0 compress.bash

### PR DESCRIPTION
**Pull request by TheHercules**

### CODE REVIEW

**Review by TheHercules — COMMENTED** [review]
Perfect? :D

File: compress.bash [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # Name of the ROM. No Spaces Please.
 # Example: CyanogenMod
 
-name=Cyanogenmod
+name=Cyanogenmod14.0
```
Comment (msfjarvis) [inline compress.bash]: Don't change the name

**Review by msfjarvis — CHANGES_REQUESTED** [review]

File: compress.bash (hunk 1 above)
Comment (TheHercules) [reply compress.bash]: fixed it. alright now?

**Comment by msfjarvis** [comment]
Will merge in a while, need to fix my server a bit

File: compress.bash (hunk 1 above)
Comment (TheHercules) [reply compress.bash]: bro..please unblock me..trust me i wont annoy you agian..ever
today is the first and last time...trust me
A sincere request, :)

**Pull request merged**